server/views/pages/summary.njk

```diff
@@ -1,46 +1,106 @@
-            <header class="app-summary-card__header">
-                <h2 class="app-summary-card__title">Personal details</h2>
-            </header>
-            <div class="app-summary-card__body">
-                {{ govukSummaryList({
-                    rows: [
-                        {
-                            key: {
-                                text: "Name"
-                            },
-                            value: {
-                                text: data.name
-                            }
-                        },
-                        {
-                            key: {
-                                text: "Gender"
-                            },
-                            value: {
-                                text: data.gender
-                            }
-                        },
-                        {
-                            key: {
-                                text: "Date of birth"
-                            },
-                            value: {
-                                text: data.dateOfBirth  | dateFormat + ' (' + data.age + ' years old)'
-                            }
-                        }
-                    ]
-                })}}
-            </div>
-        </section>
+    <h2 class="govuk-heading-l">Summary</h2>
+    <section class="app-summary-card govuk-!-margin-bottom-6 app-summary-card--large-title" id ="personal-details">
+        <header class="app-summary-card__header">
+            <h2 class="app-summary-card__title">Personal details</h2>
+        </header>
+        <div class="app-summary-card__body">
+            {{ govukSummaryList({
+                rows: [
+                    {
+                        key: {
+                        text: "Name"
+                    },
+                        value: {
+                        text: data.name
+                    }
+                    },
+                    {
+                        key: {
+                        text: "Gender"
+                    },
+                        value: {
+                        text: data.gender
+                    }
+                    },
+                    {
+                        key: {
+                        text: "Date of birth"
+                    },
+                        value: {
+                        text: data.dateOfBirth  | dateFormat + ' (' + data.age + ' years old)'
+                    }
+                    }
+                ]
+            })}}
+        </div>
+    </section>
+
+    <section class="app-summary-card govuk-!-margin-bottom-6 app-summary-card--large-title" id ="sentence">
+        <header class="app-summary-card__header">
+            <h2 class="app-summary-card__title">Sentence</h2>
+        </header>
+        <div class="app-summary-card__body">
+            <dl class="govuk-summary-list">
```

Comment:
how come you decided against using `govukSummaryList` component here? There is a html field in the value section of a row

Reply:
I decided to use html instead due to the for loops required to display the data, I'm not even sure if it's possible to use for loops inside a component tbh